arangod/Cluster/MaintenanceWorker.cpp

```diff
@@ -159,11 +159,11 @@ void MaintenanceWorker::nextState(bool actionMore) {
       _lastResult = _curAction->result();
       _curAction->endStats();
 
-      bool failed = _curAction->result().ok() && FAILED != _curAction->getState();
-      recordJobStats(failed);
+      bool ok = _curAction->result().ok() && FAILED != _curAction->getState();
+      recordJobStats(ok);
```

Comment:
As far as I can tell you just renamed `failed` to `ok`. But the logic in `recordJobStats` is reverserd, so it expects `true` on failure.

Reply:
True :scream: